weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CompareEvaluationsPage/sections/ExampleCompareSection/ExampleCompareSectionTable.tsx

```diff
@@ -1040,58 +1068,86 @@ export const ExampleCompareSectionTableModelsAsColumns: React.FC<
               );
             },
             valueGetter: (value: any, row: RowData) => {
-              return row.output[key][evaluationCallId];
+              return row.output?.[key]?.[evaluationCallId];
```

Comment:
these `?.` accessors here and in the few lines below are the only things needed for the fix - everything else is more robust scorer mismatch handling.